src/components/ResultsBody/PaginatedList/index.jsx

```diff
@@ -0,0 +1,54 @@
+import styles from "./paginatedList.module.css";
+import React, { useState, useEffect } from "react";
+import { Pagination, Box, Card, Typography } from "@mui/material";
+import { useAllVenues } from "./usePaginatedList.js";
+import { VenueCard } from "./VenueCard/index.jsx";
+
+export const PaginatedList = ({ numberOfCards }) => {
+  const itemsPerPage = numberOfCards;
```

Comment:
you should wrap all the component's logic in `usePaginatedList` hook.

Reply:
done

Reply:
done